### PR TITLE
Adding Avatar outline

### PR DESCRIPTION
**Pull request by srsharon**

### CODE REVIEW

File: module/components/Avatar/Readme.md [hunk 1]
```diff
@@ -2,8 +2,8 @@ Avatar:
 
 ```js
 <div>
-    <Avatar color="info" src='styleguide/avatar_01.jpg' /> {' '}
-    <Avatar color="info" src='styleguide/avatar_02.jpg' />
+    <Avatar color="info" src='styleguide/avatar_01.jpg' alt="Avatar" /> {' '}
+    <Avatar color="info" src='styleguide/avatar_02.jpg' alt="Avatar" />
```
Comment (jeradrutnam) [inline module/components/Avatar/Readme.md]: `alt` can be something meaningful

File: module/stylesheets/components/_avatar.scss [hunk 2]
```diff
@@ -22,4 +30,34 @@
     border-radius: 50%;
     overflow: hidden;
   }
+  &.avatar-outline-primary{
+    border: 2px solid $primary-color;
+    color: $primary-color;
+    background-color: $default-color;
```
Comment (jeradrutnam) [inline module/stylesheets/components/_avatar.scss]: I believe background color should be `transparent`

File: module/stylesheets/components/_avatar.scss [hunk 3]
```diff
@@ -22,4 +30,34 @@
     border-radius: 50%;
     overflow: hidden;
   }
+  &.avatar-outline-primary{
```
Comment (jeradrutnam) [inline module/stylesheets/components/_avatar.scss]: You can optimize this to a sass mixin and write

File: module/stylesheets/components/_avatar.scss [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
+$info-color:#17a2b8 !important;
+$warning-color:#ffc107 !important;
+$danger-color:#dc3545 !important;
+$default-color:#fff !important;
+
 .avatar {
   display: inline-flex;
   height: 50px;
   width: 50px;
+  color: #fff;
```
Comment (jeradrutnam) [inline module/stylesheets/components/_avatar.scss]: can move these values to variables

File: module/stylesheets/components/_avatar.scss [hunk 5]
```diff
@@ -1,13 +1,21 @@
+$primary-color:#007bff !important;
+$secondary-color:#868e96 !important;
+$success-color:#28a745 !important;
+$info-color:#17a2b8 !important;
+$warning-color:#ffc107 !important;
+$danger-color:#dc3545 !important;
+$default-color:#fff !important;
```
Comment (jeradrutnam) [inline module/stylesheets/components/_avatar.scss]: Move these to _variable.scss

**Review by jeradrutnam — CHANGES_REQUESTED** [review]
Please see the inline comments

File: module/components/Avatar/index.js [hunk 6]
```diff
@@ -12,48 +12,64 @@ const propTypes = {
...
+     * The src of the component. It's using the theme palette when that makes sense.
      */
     src: PropTypes.string,
     /**
-     * If `true`, the button will use outline styling.
+     * If `true`, the Avatar will use outline styling.
      */
     outline: PropTypes.bool,
+    /**
+     * If `true`, the Avatar will use outline styling.
```
Comment (jeradrutnam) [inline module/components/Avatar/index.js]: Wrong comment

File: module/components/Avatar/Readme.md [hunk 7]
```diff
@@ -2,8 +2,8 @@ Avatar:
 
 ```js
 <div>
-    <Avatar color="info" src='styleguide/avatar_01.jpg' /> {' '}
-    <Avatar color="info" src='styleguide/avatar_02.jpg' />
+    <Avatar color="info" src='styleguide/avatar_01.jpg' alt="Jane Doe" /> {' '}
+    <Avatar color="info" src='styleguide/avatar_02.jpg' alt="John Doe" />
```
Comment (jeradrutnam) [inline module/components/Avatar/Readme.md]: Why both are "John Doe" ?

File: module/components/Avatar/index.js [hunk 8]
```diff
@@ -12,48 +12,64 @@ const propTypes = {
...
+     */
+    alt: PropTypes.string,
 };
 
 const defaultProps = {
     color: 'primary',
-    outline: false
+    outline: false,
+    className: 'avatar',
```
Comment (jeradrutnam) [inline module/components/Avatar/index.js]: If you pass `avatar` as default className, there will be 2 `avatar` class names append to the element right?

**Review by jeradrutnam — COMMENTED** [review]
Please check my inline comments.

File: module/components/Avatar/Readme.md (hunk 7 above)
Comment (srsharon) [reply module/components/Avatar/Readme.md]: both are not the same one is Jane and the other is John Doe

**Review by jeradrutnam — APPROVED** [review]
Approved.

**Pull request merged**